features/git-town/configuration/parent_branch.feature

```diff
@@ -0,0 +1,34 @@
+Feature: update the parent of a nested feature branch
+
+  As a user with a nested feature branch shipped whose parent was shipped from another machine
+  I want to be able to update the parent branch for my nested feature branch
+  So that I can use it without messing with the git configuration directly
```

Comment:
We might even do this during `git sync`. If syncing a branch where the remote is gone (i.e. was there, and isn't anymore), it could change the parent to `master` (ideally after a prompt).